### PR TITLE
investment summary number formatters

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
this PR removes obsolete code from InvestmentSummary and replaces it with number formatters

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "start:translate": "(cp -n ./app/config/.env.example .env || true) && NODE_ENV=development && NF_ENABLE_TRANSLATE_OVERLAY=1 webpack-dev-server --config ./webpack/webpack.config.translate.js",
     "storybook": "STORYBOOK_RUN=1 start-storybook -p 9001 -c .storybook -s ./app/assets",
     "storybook:screenshots": "STORYBOOK_RUN=1 storybook-chrome-screenshot --debug --inject-files ./.storybook/disables.js -p 9001 -c .storybook --browser-timeout 999999 -s ./app/assets",
-    "test:browser": "TS_NODE_PROJECT=tsconfig.spec.json mocha --require ts-node/register --require jsdom-global/register --require ./test/setupBrowserTests.ts './test/setupTestsHooks.ts' './app/**/*.spec.ts?(x)'",
+    "test:browser": "TS_NODE_PROJECT=tsconfig.spec.json mocha --exit --require ts-node/register --require jsdom-global/register --require ./test/setupBrowserTests.ts './test/setupTestsHooks.ts' './app/**/*.spec.ts?(x)'",
```
Comment (Moejoe90) [inline package.json]: Please get rid of `--exit` we shouldn't use it its gonna hide any problems with hanging promises and code cleanup

**Comment by AlexeyTuboltsev** [comment]
1935

File: package.json (hunk 1 above)
Comment (AlexeyTuboltsev) [reply package.json]: the reason it hangs is `FormattedRelative` component  inside a `TimestampRow`. Obviously there's a interval or a timeout that's left hanging. It's not our code so I don't know what else is there to do

Comment (AlexeyTuboltsev) [reply package.json]: done

**Comment by AlexeyTuboltsev** [comment]
2007

File: app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx [hunk 2]
```diff
@@ -36,47 +38,141 @@ const NeuRewardCaption: React.FunctionComponent<{ isIcbm?: boolean }> = ({ isIcb
...
+}
+
+interface ITotal {
+  totalCostEur: string;
+  totalCostEth: string;
+}
+
+const EquityTokensValue: React.FunctionComponent<IEquityTockenValue> = ({ equityTokens }) => (
+  <span>
+    {/* TODO: Change to actual custom token icon (eto.equityTokenImage)*/}
```
Comment (Moejoe90) [inline app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx]: What's stopping us from doing this?

File: app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx [hunk 3]
```diff
@@ -86,32 +182,49 @@ const InvestmentTransactionDetails: TransactionDetailsComponent<ETxSenderType.IN
...
+            investmentEur={additionalData.investmentEur}
+          />
+        }
         data-test-id="invest-modal-summary-your-investment"
       />
       <InfoRow
+        data-test-id="investment-flow.summary.transaction-cost"
         caption={<FormattedMessage id="investment-flow.summary.transaction-cost" />}
-        value={`${formatEthTsd(additionalData.gasCostEth)} ETH`}
+        value={
```
Comment (Moejoe90) [inline app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx]: Really good changes already do you think it's realistic to remove to no use `MoneyNew` directly at all and have all these helper functions

File: app/components/modals/tx-sender/investment-flow/Summary.tsx [hunk 4]
```diff
@@ -21,8 +21,8 @@ interface IStateProps {
 }
 
 type IDispatchProps = {
-  onAccept: () => any;
-  onChange: () => any;
+  onAccept: () => void;
```
Comment (Moejoe90) [inline app/components/modals/tx-sender/investment-flow/Summary.tsx]: :+1:

File: intl/locales/en-en.json [hunk 5]
```diff
@@ -638,7 +638,7 @@
   "investment-flow.summary.estimated-reward": "your estimated NEU reward",
   "investment-flow.summary.eto-address": "Eto Address",
   "investment-flow.summary.token-price": "Token price",
-  "investment-flow.summary.transaction-cost": "transaction cost (gas price x gas limit)",
+  "investment-flow.summary.transaction-cost": "transaction cost (gas price × gas limit)",
```
Comment (Moejoe90) [inline intl/locales/en-en.json]: `×` is this a special character?

Comment (AlexeyTuboltsev) [reply intl/locales/en-en.json]: it's a multiplication character (U+00D7)

File: app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx (hunk 3 above)
Comment (AlexeyTuboltsev) [reply app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx]: hm, i'm not sure, there are too many variants

File: app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx (hunk 2 above)
Comment (AlexeyTuboltsev) [reply app/components/modals/tx-sender/investment-flow/InvestmentTransactionDetails.tsx]: 😁

File: intl/locales/en-en.json (hunk 5 above)
Comment (Moejoe90) [reply intl/locales/en-en.json]: :+1:

**Pull request merged**